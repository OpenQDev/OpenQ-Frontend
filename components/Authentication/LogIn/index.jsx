// Third party
import React from 'react';
import Link from 'next/link';

const LogIn = () => {
  return (
    <Link href='/login'>
      <button className={`flex justify-center btn-default whitespace-nowrap w-full`}>
        <div className='flex flex-row items-center justify-center space-x-3'>
          <div>Log in</div>
        </div>
      </button>
    </Link>
  );
};

export default LogIn;
