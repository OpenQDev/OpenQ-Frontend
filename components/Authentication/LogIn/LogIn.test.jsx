import React from 'react';
import LogIn from '.';
import renderer from 'react-test-renderer';

describe('Login Snapshot', () => {
  it('should match DOM Snapshot', () => {
    const tree = renderer.create(<LogIn />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
