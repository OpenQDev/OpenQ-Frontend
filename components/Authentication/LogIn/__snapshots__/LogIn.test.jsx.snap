// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Login Snapshot > should match DOM Snapshot 1`] = `
<a
  href="/login"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onTouchStart={[Function]}
>
  <button
    className="flex justify-center btn-default whitespace-nowrap w-full"
  >
    <div
      className="flex flex-row items-center justify-center space-x-3"
    >
      <div>
        Log in
      </div>
    </div>
  </button>
</a>
`;
